test_settings.py

```diff
@@ -43,3 +43,9 @@ def root(*args):
 
 SECRET_KEY = 'insecure-secret-key'
 
+TEMPLATES = [
```

Comment:
This is conflicting with the current master, please rebase.

Reply:
Rebased